projects/fsastorefrontlib/src/shared/util/helpers/sortByProductName.pipe.ts

```diff
@@ -0,0 +1,9 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { OrderEntry } from '@spartacus/core';
+
+@Pipe({ name: 'cxSortByProductName' })
+export class SortByProductNamePipe implements PipeTransform {
+  transform(input: OrderEntry[]): any {
+    return input.sort((a, b) => a.product.name.localeCompare(b.product.name));
```

Comment:
Can we try to implement more generic function that can be used for most cases? Like `input.sort((a, b) => a-b);`? This may require one more additional `ng-container` with `ng-for` for `product.name` in HTML template (in this example), but I think it's worth of shot...

Reply:
Strings can't be compared like that I think. For html I don't have idea how to restructure it and pass just product name to generic method because it's nested and for sorting products whole order entry array is needed.
At least I could put configurable parameters for nested fields.

Reply:
Yes, but in sort method it should work, numbers come before strings.